src/subdomains/core/route/route.entity.ts

```diff
@@ -0,0 +1,37 @@
+import { IEntity } from 'src/shared/models/entity';
+import { UserData } from 'src/subdomains/generic/user/models/user-data/user-data.entity';
+import { User } from 'src/subdomains/generic/user/models/user/user.entity';
+import { Column, Entity, OneToOne } from 'typeorm';
+import { Buy } from '../buy-crypto/routes/buy/buy.entity';
+import { Swap } from '../buy-crypto/routes/swap/swap.entity';
+import { Sell } from '../sell-crypto/route/sell.entity';
+
+@Entity()
+export class Route extends IEntity {
+  @Column({ length: 256, unique: true, nullable: true })
+  label: string;
```

Comment:
I think this unique index is not helpful/required?

Reply:
That's what the ticket says or? 🤔 
Robin said in the ticket, the label col has to be unique on the complete table

Reply:
Ah, true, you're right